test/regression/2016-11-24-bitcast-weak-alias.c

```diff
@@ -1,3 +1,4 @@
+// REQUIRES: linux
```

Comment:
@andreamattavelli This is not correct. Currently this will disable the test **everywhere**.
The `REQUIRES:` directive looks at the `config.available_features` in `test/lit.cfg`. There is no `linux` feature so conditional logic needs to be added to add the feature if `lit` is being run on Linux.